src/Sarif/Sarif.csproj

```diff
@@ -18,14 +18,18 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
You want net45 here as well for the SDK library itself, right?

Reply:
Yes, added this back.

---
In reply to: [391735210](https://github.com/microsoft/sarif-sdk/pull/1812#discussion_r391735210) [](ancestors = 391735210)